easybuild/easyblocks/generic/intelbase.py

```diff
@@ -325,11 +324,11 @@ def install_step(self, silent_cfg_names_map=None, silent_cfg_extras=None):
 
         if self.install_components:
             if len(self.install_components) == 1:
-                # if ALL or DEFAULTS, no quotes should be used
+                # if only a single components if listed (e.g. ALL, DEFAULTS), no quotes should be used
```

Comment:
sure, but that doesn't mean we shouldn't have a proper check here, people may *try* it, and they shouldn't run into a weird crash